src/data/markdown/docs/02 javascript api/08 k6-http/60 CookieJar/CookieJar-clear-url.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'CookieJar.clear(url)'
+excerpt: 'Delete all cookies for the given URL.'
+---
+
+| Parameter | Type   | Description |
+| --------- | ------ | ----------- |
+| url       | string | The URL for which to delete all cookies. |
+
+### Example
+
+<CodeGroup labels={[]}>
+
+```javascript
+import http from 'k6/http';
+
+export default function () {
+    http.get('https://httpbin.test.k6.io/cookies/set?one=1&two=2');
+
+    // We'll use httpbin's reflection to see what cookies we
+    // are actually sending to the server after every change
+    let httpbinResp;
+    httpbinResp = http.get('https://httpbin.test.k6.io/cookies');
+    console.log(JSON.stringify(httpbinResp.json().cookies));
+    // Will print '{"one":"1","two":"2"}'
+
+    const jar = http.cookieJar();  // get the VU specific jar
+    jar.set('https://httpbin.test.k6.io/cookies', 'three', '3');
+    httpbinResp = http.get('https://httpbin.test.k6.io/cookies');
+    console.log(JSON.stringify(httpbinResp.json().cookies));
+    // Will print '{"one":"1","three":"3","two":"2"}'
+
+    jar.delete('https://httpbin.test.k6.io/cookies', 'one');
+    httpbinResp = http.get('https://httpbin.test.k6.io/cookies');
+    console.log(JSON.stringify(httpbinResp.json().cookies));
+    // Will print '{"three":"3","two":"2"}'
+
+    jar.clear('https://httpbin.test.k6.io/cookies');
+    httpbinResp = http.get('https://httpbin.test.k6.io/cookies');
+    console.log(JSON.stringify(httpbinResp.json().cookies));
+    // Will print '{}'
+}
+```
```

Comment:
I would move this example to the main page instead of copying it in both `delete` and `clear`

Reply:
the main page has a slightly different example :man_shrugging: and I copied it in both places, since it illustrates the difference between both